cmd/helm/helm_test.go

```diff
@@ -97,6 +97,8 @@ func storageFixture() *storage.Storage {
 	return storage.Init(driver.NewMemory())
 }
 
+// As of Git commit 598dc504d634332f955f5e984d1d53696ba07abc (Web Feb 19, 2020), go-shellwords seems to have been fixed
```

Comment:
can you remove this comment as well as the function that was commented out?

Reply:
done